sessions/recap-project-6/tourio-app/pages/api/places/[id]/index.js

```diff
@@ -7,11 +8,16 @@ export default function handler(request, response) {
     return;
   }
 
-  const place = places.find((place) => place.id === id);
+  const place = db_places.find((place) => place._id.$oid === id);
+  const comment = place?.comments;
+  const allCommentIds = comment?.map((comment) => comment.$oid) || [];
+  const comments = db_comments.filter((comment) =>
+    allCommentIds.includes(comment._id.$oid)
+  );
```

Comment:
So you implemented the comments in the "non db" version as well. In my opinion this can confuse the students a lot. From my side the starter template should not include any elements regarding the comments (except the Comments Component maybe). By this we don't need the comments included in the data from the beginning, preventing the need to the awkward Object Ids. 
I would suggest that we keep your example solution and the starter template from the old project. We could think about providing example data with the example solution / bonus part to get the students going. They could replace their current data in their DB with the new data with comments as soon as they reach the bonus challenge. In this way, the students can focus on the first tasks and can shift towards the bonus challenge if they want to later.